### PR TITLE
WIP - elasticsearch REST storage component

### PR DESCRIPTION
**Pull request by devinsba**
#1253

### CODE REVIEW

File: zipkin-storage/elasticsearch-rest/pom.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>rest</artifactId>
+            <version>5.0.0-alpha5</version>
+        </dependency>
+
+        <dependency>
+            <groupId>io.zipkin.java</groupId>
```
Comment (codefromthecrypt) [inline zipkin-storage/elasticsearch-rest/pom.xml:45]: note: I'm not sure elasticsearch 5 has a guava dep anymore..

**Comment by codefromthecrypt** [comment]
thanks for starting on this.. I got sidetracked, but understand this is a big deal, not just AWS, but also for ElasticSearch 5. I'll try to dedicate some time towards this, as we really need a reliable and performant backend that most people can use.

**Comment by devinsba** [comment]
Awesome. I might get some time for this today too if you need a rubber duck for anything

On Tue, Sep 6, 2016 at 11:28 PM -0400, "Adrian Cole" notifications@github.com wrote:

thanks for starting on this.. I got sidetracked, but understand this is a big deal, not just AWS, but also for ElasticSearch 5. I'll try to dedicate some time towards this, as we really need a reliable and performant backend that most people can use.

**Comment by codefromthecrypt** [comment]
with my current backlog, I think earliest I can start is this weekend.

**Comment by codefromthecrypt** [comment]
ok I'll start on this now. Similar to you, I think I'll start with the collector side.

**Comment by codefromthecrypt** [comment]
thanks for the tips. I started this on a shared branch https://github.com/openzipkin/zipkin/pull/1294

**Pull request closed without merging**